src/main/resources/application.properties

```diff
@@ -40,6 +40,13 @@ auth.provider.service.client.microservice=ccd_data
 auth.provider.service.client.key=${DATA_STORE_IDAM_KEY:AAAAAAAAAAAAAAAB}
```

Comment:
The old S2S properties should be removed as they are not used anymore.

Reply:
OK, I'll remove all four `auth.provider.service.client.*` properties.